specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2022-09-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-01",
+    "title": "AppServiceCertificateOrders API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/certificateOrders": {
+      "get": {
+        "tags": [
+          "AppServiceCertificateOrders"
+        ],
+        "summary": "List all certificate orders in a subscription.",
+        "description": "Description for List all certificate orders in a subscription.",
+        "operationId": "AppServiceCertificateOrders_List",
```

Comment:
There's no obvious limit to how many orders there can be on a sub - are you going to support paged or filtered requests with take, skip, filter etc params?